synapse/cores/common.py

```diff
@@ -259,67 +297,54 @@ def setModlVers(self, name, vers):
             self.setRowsByIdProp(iden, prop, vers)
             return vers
 
-    def revModlVers(self, name, revs):
-        '''
-        Update a model using a list of (vers,func) tuples.
+    def _addDataModels(self, modtups):
 
-        Args:
-            name (str): The name of the model
-            revs ([(int,function)]):  List of (vers,func) revision tuples.
+        DataModel._addDataModels(self, modtups)
```

Comment:
The Cortex inherits from DataModel already - is there a reason to override this function name and do this call the way we are doing it?

Reply:
we're over-riding a parent class method so we can do some initialization as well.  we need to call his so he can initialize as well....

Reply:
:+1: